site/src/assets/docs-content/android/configuration.json

```diff
@@ -1 +1 @@
n-icon>\n  Changing App Name\n  </a>\n</h2>\n<p>To change the name of your app, change the value for <code>app_name</code> in <code>strings.xml</code>:</p>\n\n  <highlight-code-line >\n    <pre class=\"language-xml\"><code class=\"language-xml\">&lt;string name=\"app_name\"&gt;MyApp&lt;/string&gt;</code></pre>\n  </highlight-code-line>\n  <p>You probably also want to set the Activity name to match the App, for apps that plan to only have one activity (the main web activity running your app):</p>\n\n  <highlight-code-line >\n    <pre class=\"language-xml\"><code class=\"language-xml\">&lt;string name=\"title_activity_main\"&gt;MyApp&lt;/string&gt;</code></pre>\n  </highlight-code-line>\n  \n<h2 id=\"changing-custom-url\">\n  <a class=\"heading-link\" href=\"#changing-custom-url\"><ion-icon name=\"ios-link\"></ion-icon>\n  Changing Custom URL\n  </a>\n</h2>\n<p>Your app can respond to custom URLs on launch, making it possible to handle deeplinks and app interactions.</p>\n<p>To change the URL, search for and modify this line in <code>strings.xml</code>. It&#39;s recommended to set this to the bundle/app id.</p>\n\n  <highlight-code-line >\n    <pre class=\"language-xml\"><code class=\"language-xml\">&lt;string name=\"custom_url_scheme\"&gt;com.getcapacitor.myapp&lt;/string&gt;</code></pre>\n  </highlight-code-line>\n  <p>In this example, the app will respond to URLs with the <code>com.getcapacitor.myapp://</code> scheme.</p>\n\n<h2 id=\"setting-permissions\">\n  <a class=\"heading-link\" href=\"#setting-permissions\"><ion-icon name=\"ios-link\"></ion-icon>\n  Setting Permissions\n  </a>\n</h2>\n<p>In Android, permissions your app will need are defined in <code>AndroidManifest.xml</code> inside of the <code>&lt;manifest&gt;</code> tag, generally at the bottom\nof the file.</p>\n<p>For example, here&#39;s what adding Network permissions looks like:</p>\n\n  <highlight-code-line >\n    <pre class=\"language-xml\"><code class=\"language-xml\">&lt;manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\npackage=\"com.getcapacitor.myapp\"&gt;\n    &lt;activity&gt;\n      &lt;!-- other stuff --&gt;\n    &lt;/activity&gt;\n\n    &lt;!-- More stuff --&gt;\n\n    &lt;!-- Your permissions --&gt;\n\n    &lt;!-- Network API --&gt;\n    &lt;uses-permission android:name=\"android.permission.ACCESS_NETWORK_STATE\" /&gt;\n&lt;/manifest&gt;</code></pre>\n  </highlight-code-line>\n  <p>Generally, the plugin you choose to use will ask you to set a permission. Add it in this file.</p>\n\n<h2 id=\"default-permissions\">\n  <a class=\"heading-link\" href=\"#default-permissions\"><ion-icon name=\"ios-link\"></ion-icon>\n  Default Permissions\n  </a>\n</h2>\n<p>By default, the entire initial permissions requested for the latest version of Capacitor with the standard plugins can be found in the android-template&#39;s <a href=\"https://github.com/ionic-team/capacitor/blob/master/android-template/app/src/main/AndroidManifest.xml\">AndroidManifest.xml</a></p>\n"}
```

Comment:
this and the other files were auto updated when i ran "npm run start" locally...